app/Http/Controllers/AssetModelsController.php

```diff
@@ -141,15 +151,15 @@ public function store()
         $model = new AssetModel;
 
         $settings=Input::all();
-        $settings['eol']= null;
+        $settings['eol']=0;
```

Comment:
Why are you setting this to 0? Should be null if no EOL is given. (We can't assume 0)